Kitodo-DataAccess/src/main/java/org/kitodo/dataaccess/SerializationFormat.java

```
@@ -0,0 +1,170 @@
+     */
+    RDF_XML_ABBREV {
+        @Override
+        public void write(Node node, Map<String, String> map, File file) throws FileNotFoundException {
+            write(node, map, file, "RDF/XML-ABBREV");
+        }
+    },
+    /**
+     * The Turtle serialization format ({@code text/turtle}).
+     *
+     * @see "https://en.wikipedia.org/wiki/Turtle_(syntax)"
+     */
+    TURTLE {
+        @Override
+        public void write(Node node, Map<String, String> map, File file) throws FileNotFoundException {
+            write(node, map, file, "TURTLE");
+        }
+    };
+
+    /**
+     * Write a serialised representation of this model in a specified language.
+     *
+     * @param file
+     *            a file to write to
+     * @param map
+     *            user defined namespace prefixes, mapped from prefix to
+     *            namespace, the namespace must end either in {@code #} or
+     *            {@code /}
+     * @param lang
+     *            the language in which to write the model, predefined values
+     *            are {@code RDF/XML}, {@code RDF/XML-ABBREV}, {@code N-TRIPLE},
+     *            {@code TURTLE} and {@code N3}.
+     */
+    static void write(Node node, Map<String, String> map, File file, String lang) throws FileNotFoundException {
+        FileOutputStream fos = null;
+        BufferedOutputStream bos = null;
+        try {
+            fos = new FileOutputStream(file);
+            bos = new BufferedOutputStream(fos);
+            write(node, map, bos, lang);
+        } finally {
+            if (bos != null) {
+                try {
+                    bos.close();
+                } catch (IOException e) {
+                    /* close silently */
+                }
+            }
+            if (fos != null) {
+                try {
+                    fos.close();
+                } catch (IOException e) {
+                    /* close silently */
```

no empty catch blocks, at least logger